openmetadata-ui/src/main/resources/ui/src/constants/explore.constants.ts

```diff
@@ -11,319 +11,105 @@
  *  limitations under the License.
  */
 
-import { toLower } from 'lodash';
-import { AggregationType, Bucket, FilterObject } from 'Models';
-import { TabCounts } from '../components/Explore/explore.interface';
-import { EntityType } from '../enums/entity.enum';
+import { isNil } from 'lodash';
+import { FilterObject } from '../components/AdvancedSearch/AdvancedSearch.interface';
+import { ExploreSearchIndex } from '../components/Explore/explore.interface';
+import { SortingField } from '../components/Explore/SortingDropDown';
 import { SearchIndex } from '../enums/search.enum';
-import { getFilterKey } from '../utils/FilterUtils';
 import { Icons } from '../utils/SvgUtils';
-import { tiers } from './constants';
 
-export const INITIAL_SORT_FIELD = 'updatedAt';
+/**
+ * Builds an Elasticsearch JSON query from a {@see FilterObject}
+ */
+export const filterObjectToElasticsearchQuery: (
+  f: FilterObject | undefined
+) => Record<string, unknown> | undefined = (f) =>
+  isNil(f)
+    ? { query: { bool: {} } }
+    : Object.entries(f).length
+    ? {
+        query: {
+          bool: {
+            must: Object.entries(f).map(([key, values]) => ({
+              bool: {
+                should: values.map((value) => ({ term: { [key]: value } })),
+              },
+            })),
+          },
+        },
+      }
+    : undefined;
+
```

Comment:
Move this to utils and advice to reduce double ternary to normal if/else to improve code readability